app/javascript/react/containers/ReviewsContainer.js

```diff
@@ -8,7 +8,6 @@ class ReviewsContainer extends Component {
       reviews: []
     }
   }
-
   componentDidMount() {
     fetch(`/api/v1/parks/${this.props.park_id}/reviews`)
```

Comment:
Oh little white space, how I miss thee,
for thine breadth was a memory, 
of distinction and divide,
And without your just deliverance,
I'll surely not abide. 

potato